capture_service/dive_client_cli.cc

```diff
@@ -26,103 +28,192 @@ limitations under the License.
 #include "absl/flags/internal/flag.h"
 #include "absl/flags/parse.h"
 #include "absl/flags/usage.h"
+#include "absl/strings/str_cat.h"
 #include "absl/strings/str_format.h"
 #include "absl/strings/str_split.h"
+
 #include "android_application.h"
 #include "command_utils.h"
 #include "constants.h"
 #include "device_mgr.h"
 #include "network/tcp_client.h"
-#include "absl/strings/str_cat.h"
 
 using namespace std::chrono_literals;
 
-enum class Command
-{
-    kNone,
-    kGfxrCapture,
-    kGfxrReplay,
-    kListDevice,
-    kListPackage,
-    kRunPackage,
-    kRunAndCapture,
-    kCleanup,
-};
-
-bool AbslParseFlag(absl::string_view text, Command* command, std::string* error)
+absl::Status ValidateRunOptions(const GlobalOptions& options)
 {
-    if (text == "list_device")
+    if (options.package.empty() && options.vulkan_command.empty())
     {
-        *command = Command::kListDevice;
-        return true;
+        return absl::InvalidArgumentError("Missing required flag: --package or --vulkan_command");
     }
-    if (text == "list_package")
+
+    static const std::set<std::string> valid_types = { "openxr", "vulkan", "vulkan_cli" };
```

Comment:
maybe better to be consistent with https://github.com/google/dive/pull/515/files#r2528100218

Reply:
sorry a bit confused here, aren't we supposed to support following 4 types?
`vulkan_openxr, vulkan_non_openxr, vulkan_cli_non_openxr, gles_openxr`